### PR TITLE
Non descriptive links index

### PR DESCRIPTION
**Pull request by david-kasali**

### CODE REVIEW

File: public/static/locales/cy/common.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
...
+  "is the local authority your registration will be sent to. Is this the correct local authority for your business? If unsure please use this [Food Business Registration link where you can check for your correct local authority](www.gov.uk/food-business-registration) using the location or site of your food business.": "yw'r awdurdod lleol y bydd eich cofrestriad yn cael ei anfon ato. Ai hwn yw'r awdurdod lleol cywir ar gyfer eich busnes? Os ydych chi'n ansicr, defnyddiwch y [ddolen Cofrestru Busnes Bwyd lle gallwch wirio am eich awdurdod lleol cywir](https://www.gov.uk/food-business-registration) gan ddefnyddio lleoliad neu safle eich busnes bwyd.",
```
Comment (paul-burnett) [inline public/static/locales/cy/common.json:91]: Why is this added instead of updating the old one above?

Comment (david-kasali) [reply public/static/locales/cy/common.json:90]: removed the initial one now

**Review by paul-burnett — APPROVED** [review]

**Comment by paul-burnett** [comment]
Closing PR as changes incorporated into frontend migration
